src/components/header.component.vue

```diff
@@ -0,0 +1,395 @@
+<template>
+  <div class="HeaderWrapper">
```

Comment:
클래스명은 케밥케이스로 작성하는게 권장되는 사항입니다
구형 브라우저에서 html상에 쓰여진 문자를 모두 소문자로 인식한다고 알고 있고, Vue에서 자식 컴포넌트에 props를 전달할 때에도 실제 변수에 대문자가 들어가도 대시(-)로 치환하여 내려주게 됩니다.
```suggestion
  <div class="header-wrapper">
```